### ISSUE TITLE
Some fields in new incident page don't work.

### ISSUE THREAD

**Issue by wsanchez** [burningmantech/ranger-ims-server#176]
If you start by editing summary, priority, state, or add a report entry, a new incident gets created.

If you instead edit Rangers, incident types, location, attached reports first, the page gets screwy.  These all involved relational data (in the case of location, it's the concentric streets), so it's probably got something to do with that.

If this proves hard to fix, a work-around could be to disable those fields until the incident gets created in the DB.

**Comment by wsanchez** [comment]
Hrm.  Editing Rangers and incident types in a new problem is working now, so it's not "anything with a relational table" like I thought.

**Comment by wsanchez** [comment]
If I edit address radial hour first, I get an error.  Server side:

```
2017-08-11T08:52:16-0700 [ims.application._klein#critical] Request failed
	Traceback (most recent call last):
	  File "/Users/wsanchez/Dropbox/Developer/BurningMan/ranger-ims-server/.tox/run/lib/python3.6/site-packages/klein/app.py", line 126, in execute_endpoint
	    return endpoint_f(self._instance, *args, **kwargs)
	  File "/Users/wsanchez/Dropbox/Developer/BurningMan/ranger-ims-server/.tox/run/lib/python3.6/site-packages/klein/app.py", line 211, in _f
	    return _call(instance, f, request, *a, **kw)
	  File "/Users/wsanchez/Dropbox/Developer/BurningMan/ranger-ims-server/.tox/run/lib/python3.6/site-packages/klein/app.py", line 51, in _call
	    result = ensureDeferred(result)
	  File "/Users/wsanchez/Dropbox/Developer/BurningMan/ranger-ims-server/.tox/run/lib/python3.6/site-packages/twisted/internet/defer.py", line 908, in ensureDeferred
	    return _inlineCallbacks(None, coro, Deferred())
	--- <exception caught here> ---
	  File "/Users/wsanchez/Dropbox/Developer/BurningMan/ranger-ims-server/.tox/run/lib/python3.6/site-packages/twisted/internet/defer.py", line 1386, in _inlineCallbacks
	    result = g.send(result)
	  File "/Users/wsanchez/Dropbox/Developer/BurningMan/ranger-ims-server/.tox/run/lib/python3.6/site-packages/ims/application/_api.py", line 328, in newIncidentResource
	    incident = modelObjectFromJSONObject(json, Incident)
	  File "/Users/wsanchez/Dropbox/Developer/BurningMan/ranger-ims-server/.tox/run/lib/python3.6/site-packages/ims/model/json/_json.py", line 61, in modelObjectFromJSONObject
	    return jsonDeserialize(object, modelClass)
	  File "/Users/wsanchez/Dropbox/Developer/BurningMan/ranger-ims-server/.tox/run/lib/python3.6/site-packages/cattr/converters.py", line 138, in structure
	    if not isinstance(cl, _Union)
	  File "/Users/wsanchez/Dropbox/Developer/BurningMan/ranger-ims-server/.tox/run/lib/python3.6/site-packages/cattr/converters.py", line 130, in <lambda>
	    self._structure.register(cl, lambda t, o: func(o, t))
	  File "/Users/wsanchez/Dropbox/Developer/BurningMan/ranger-ims-server/.tox/run/lib/python3.6/site-packages/ims/model/json/_incident.py", line 218, in deserializeIncident
	    for key in IncidentJSONKey
	  File "/Users/wsanchez/Dropbox/Developer/BurningMan/ranger-ims-server/.tox/run/lib/python3.6/site-packages/ims/model/json/_incident.py", line 218, in <genexpr>
	    for key in IncidentJSONKey
	  File "/Users/wsanchez/Dropbox/Developer/BurningMan/ranger-ims-server/.tox/run/lib/python3.6/site-packages/cattr/converters.py", line 138, in structure
	    if not isinstance(cl, _Union)
	  File "/Users/wsanchez/Dropbox/Developer/BurningMan/ranger-ims-server/.tox/run/lib/python3.6/site-packages/cattr/converters.py", line 130, in <lambda>
	    self._structure.register(cl, lambda t, o: func(o, t))
	  File "/Users/wsanchez/Dropbox/Developer/BurningMan/ranger-ims-server/.tox/run/lib/python3.6/site-packages/ims/model/json/_location.py", line 85, in deserializeLocation
	    name=obj[LocationJSONKey.name.value],
	builtins.KeyError: 'name'
```

**Comment by wsanchez** [comment]
OK, it looks like any location field produces a JSON-related error.  All other fields are OK.

**Comment by wsanchez** [comment]
Fixed in #183
